clients/bazel-module-registry/src/main/kotlin/RemoteBazelModuleRegistryService.kt

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.clients.bazelmoduleregistry
+
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonNamingStrategy
+
+import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.OkHttpClient
+
+import retrofit2.Retrofit
+import retrofit2.converter.kotlinx.serialization.asConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+/**
+ * The JSON (de-)serialization object used by this client.
+ */
+private val JSON = Json {
+    ignoreUnknownKeys = true
+    namingStrategy = JsonNamingStrategy.SnakeCase
+}
+
+/**
+ * The client uses the Bazel Central Registry by default.
+ */
+private const val DEFAULT_URL = "https://bcr.bazel.build"
+
+/**
+ * Interface for a Bazel Module Registry, based on the directory structure of https://bcr.bazel.build/ and the Git
+ * repository it is based on (https://github.com/bazelbuild/bazel-central-registry/).
+ */
+interface RemoteBazelModuleRegistryService : BazelModuleRegistryService {
```

Comment:
Could you please to the pure renaming of this class / file in a preceding commit ("refactor(bazel)!: Rename ... to prepare for ...") to ease the commit-by-commit review?